### ISSUE TITLE
The result of 3D perception using the BEVDepth method

### ISSUE THREAD

**Issue by zx-repo** [jinlong17/LightDiff#10]
Thank you for your work! But I have some confusion.
As described in the paper,  I used the only-front camera images of 616 daytime scenes and 15 nighttime scenes in nuScenes as the training set and the test set, respectively, with resnet-50 as the backbone for 24 epochs of training. Then the result I got was quite different from the paper
Could you explain the training configuration and the dataset you use in detail?
![屏幕截图 2024-11-01 131127](https://github.com/user-attachments/assets/1d8f8599-f984-4e31-99ca-5f841cb9c791)
